MANIFEST.in

```diff
@@ -0,0 +1,44 @@
+include CHANGELOG.md
+include Doxyfile
+include LICENSE
+include Makefile
+include NOTICE
+include README.md
+include rcfile.pylint
+include requirements.txt
+include requirements-dev.txt
+include setup.py
+include tox.ini
+include tools/hilbert
+include tools/hilbert.py
+include hilbert_config/__init__.py
+include hilbert_config/helpers.py
+include hilbert_config/hilbert_cli_config.py
+include hilbert_config/subcmdparser.py
+include tests/__init__.py
```

Comment:
`MANIFEST.in` is for packaging, right?
- I think it is possible to use wildcards instead of listing every possible file

Reply:
yes, it supports wildcards, e.g. `include some_dir/*.py` but i was reluctant to use them since there there are some files unrelated to the package